storage/storage.go

```diff
@@ -1810,6 +1812,27 @@ func (o *ObjectHandle) Retryer(opts ...RetryOption) *ObjectHandle {
 	return &o2
 }
 
+// Retryer returns an Client that is configured with custom retry behavior as
+// specified by the options that are passed to it. All operations on the new
+// handle will use the customized retry configuration.
+// Retry options set on a bucket or object handle will take precedence over
+// these options.
+func (c *Client) Retryer(opts ...RetryOption) *Client {
```

Comment:
Let's not have this return a new client, but instead configure the existing one. We can call it `SetRetry` maybe.

I used that pattern on BucketHandle/ObjectHandle because there were existing methods that used the same pattern (e.g. `If`, `Key`, `Generation`). But client methods generally don't follow the same pattern. And we want users to set this once before using the client if they plan on using it, not setting it using chained calls while calling a method.

Reply:
Good note